app/controllers/providers/check_merits_answers_controller.rb

```diff
@@ -8,7 +8,7 @@ def show
 
     def continue
       unless draft_selected?
-        legal_aid_application.generate_reports! if legal_aid_application.may_generate_reports?
+        legal_aid_application.generate_reports!
```

Comment:
If we are removing this, can we remove:
app/models/legal_aid_application.rb:96 ?